docs/docs/api/gesture-handlers/rotation-gh.md

```diff
@@ -1,43 +1,44 @@
 ---
-id: rotation-gh
+id: handler-rotation
 title: RotationGestureHandler
-sidebar_label: Rotation
+sidebar_label: RotationGestureHandler
```

Comment:
[I recently changed](https://github.com/software-mansion/react-native-gesture-handler/pull/1329) ids and titles in the sidebar, so this should be reverted.

Reply:
Please also revert id change.